framework/platform/bpl/include/bpl/bpl_cfg.h

```diff
@@ -381,9 +389,10 @@ int cfg_get_beerocks_credentials(const int radio_dir, char ssid[BPL_SSID_LEN],
  * the wpa-supplicant will get from there in runtime, without supply it on `connect` API, or not.
  *
  * @return mem_only_psk flag on success.
+ * @return default_value if value not set
  * @return -1 Error.
  */
-int cfg_get_security_policy();
+int cfg_get_security_policy(int default_value = -1);
```

Comment:
Not sure about this one - @kantera800 ?

Reply:
If we're not sure, leave it as no default for the time being.